src/scrape_up/wuzzuf/stringdecorator.py

```diff
@@ -0,0 +1,39 @@
+def string_decorator(func):
```

Comment:
No need of this, just return None in any case the scraper is not able to return the result.
We do create files only for modules.